Update SSO instructions for new options (#7481)
fishtown-analytics/docs.getdbt.com: website/docs/docs/cloud/manage-access/set-up-sso-saml-2.0.md

      </Expandable>
      <!-- vale on -->

### Additional configuration options

> 

Reviewer: Same feedback as above, can we lead in with a brief, single sentence lead in that describes where these config options are located?

> **Sign SAML AuthnRequest** will cause <Constant name="cloud" /> to sign SAML reqeusts sent to your identity provider when users attempt to login.  Metadata for configuring this in your identity provider can be downloaded from the value shown in **SAML Metadata URL**.  Leaving this disabled is recommended for most situations.

Reviewer: ```suggestion
- **Sign SAML Auth Request:** <Constant name="cloud" /> will sign SAML requests sent to your identity provider when users attempt to log in.  Metadata for configuring this in your identity provider can be downloaded from the value shown in **SAML Metadata URL**.  We recommend leaving this disabled for most situations.
```

> **Attribute Mappings** provides a way to associate SAML attributes which <Constant name="cloud" /> needs with attributes your identity provider includes in SAML assertions.  The value must be a valid JSON object with keys of `email`, `first_name`, or `last_name` and values that are strings or lists of strings.  For example, if your identity provider is unable to include an 'email' attribute in assertions, but does include one called `EmailAddress`, then **Attribute Mappings** should be set to `{ "email": "EmailAddress" }`.  These mappings are only needed if you cannot configure attributes as specified in the instructions above.  If that can be done, the default value of `{}` is acceptable.

Author: ```suggestion
**Attribute Mappings** provides a way to associate SAML attributes which <Constant name="cloud" /> needs with attributes your identity provider includes in SAML assertions.  The value must be a valid JSON object with keys of `email`, `first_name`, or `last_name` and values that are strings or lists of strings.  For example, if your identity provider is unable to include an `email` attribute in assertions, but does include one called `EmailAddress`, then **Attribute Mappings** should be set to `{ "email": "EmailAddress" }`.  These mappings are only needed if you cannot configure attributes as specified in the instructions above.  If that can be done, the default value of `{}` is acceptable.
```

> **Attribute Mappings** provides a way to associate SAML attributes which <Constant name="cloud" /> needs with attributes your identity provider includes in SAML assertions.  The value must be a valid JSON object with keys of `email`, `first_name`, or `last_name` and values that are strings or lists of strings.  For example, if your identity provider is unable to include an `email` attribute in assertions, but does include one called `EmailAddress`, then **Attribute Mappings** should be set to `{ "email": "EmailAddress" }`.  These mappings are only needed if you cannot configure attributes as specified in the instructions above.  If that can be done, the default value of `{}` is acceptable.

Reviewer: ```suggestion
- **Attribute Mappings:** Associate SAML attributes that <Constant name="cloud" /> needs with attributes your identity provider includes in SAML assertions.  The value must be a valid JSON object with the `email`, `first_name`, or `last_name` keys and values that are strings or lists of strings.  For example, if your identity provider is unable to include an `email` attribute in assertions, but does include one called `EmailAddress`, then **Attribute Mappings** should be set to `{ "email": "EmailAddress" }`. The mappings are only needed if you cannot configure attributes as specified in the instructions on this page. If you can, the default value of `{}` is acceptable.
```
